ecl/hqlcpp/hqlttcpp.cpp

```diff
@@ -6236,7 +6236,13 @@ IHqlExpression * WorkflowTransformer::transformInternalFunction(IHqlExpression *
         ForEachChild(idx, bodyCode)
         {
             IHqlExpression *child = bodyCode->queryChild(idx);
-            if (child->isAttribute() && child->queryName() != languageAtom && child->queryName() != importAtom)
+            if (child->isAttribute() &&
+                child->queryName() != languageAtom &&
```

Comment:
minor:  I would normally common up child->queryName(), possibly by extracting this into a isStrippedAttribute(IAtom *)